migrate/migrations/20210518145011-webhook-user-contact-method-type.sql

```diff
@@ -0,0 +1,6 @@
+-- +migrate Up notransaction
+
+ALTER TYPE enum_user_contact_method_type ADD VALUE IF NOT EXISTS 'WEBHOOK';
+
+-- +migrate Down
+
```

Comment:
Missing removal of type in migrate down

Reply:
We should probably do that then, so if we have to rollback this isn't left in the DB even though the feature doesn't exist

Reply:
A while back we switched to just leaving these in going forward; re-creating and swapping the type has other potential side-effects that weren't worth the risk. The migration test has a special case to ignore this specifically:
```go
// enumOK handles checking for safe enum differences. This case is that migrate
// up can add, but migrate down will not remove new enum values.
//
// migrate down can't safely remove enum values, but it's safe for new ones
// to exist. So we simply check that all original items exist.
func enumOK(got, want string) bool {
```